source/android/adaptivecards/src/androidTest/java/io/adaptivecards/objectmodel/DateInputPropertiesTest.java

```diff
@@ -194,6 +199,103 @@ public DateInput castTo(BaseCardElement element) {
         }
     }
 
+    @Test
+    public void MinValidationTest() throws Exception
+    {
+        DateInput dateInput = TestUtil.createMockDateInput();
+        dateInput.SetMin(c_inRangeDateValues[0]);
+
+        DateInputHandler dateInputHandler = new DateInputHandler(dateInput, null);
+        dateInputHandler.setView(new EditText(InstrumentationRegistry.getContext()));
+
+        // Validate that empty input is always valid
+        Assert.assertEquals(true, dateInputHandler.isValid());
+
+        // Validate only min
+        TestUtil.SpecificsValidationExecutor specsExecutor = new TestUtil.SpecificsValidationExecutor(dateInputHandler);
+        TestUtil.runValidationTests(c_smallDateValues, false, specsExecutor);
+        TestUtil.runValidationTests(TestUtil.concat(String.class, c_inRangeDateValues, c_largeDateValues), true, specsExecutor);
+
+        // Validate non-required + min
+        TestUtil.GeneralValidationExecutor gralExecutor = new TestUtil.GeneralValidationExecutor(dateInputHandler);
+        TestUtil.runValidationTests(c_smallDateValues, false, gralExecutor);
```

Comment:
I hope we aren't using Hungarian in our code? this shud ideally be SmallDateValues or smallDateValues depending on the preferred casing for constants in java.

Reply:
we are using Hungarian in the shared model, uwp and android. This comes from the very first version of AdaptiveCards